Sources/PT.PM.CSharpParseTreeUst/Converter/ExpressionVisitor.cs

```diff
@@ -73,6 +73,36 @@ public override Ust VisitAnonymousObjectCreationExpression(AnonymousObjectCreati
             return result;
         }
 
+        public override Ust VisitTupleExpression(TupleExpressionSyntax node)
+        {
+            var result = new TupleCreateExpression()
+            {
+                TextSpan = node.GetTextSpan(),
+                Initializers = new List<Expression>()
+            };
+
+            for (int i = 0; i < node.Arguments.Count; i++)
+            {
+                var arg = node.Arguments[i];
+                var assignment = new AssignmentExpression
+                {
+                    Right = (Expression)VisitAndReturnNullIfError(arg.Expression),
+                    TextSpan = arg.GetTextSpan()
+                };
+                if (arg.NameColon == null)
+                {
+                    assignment.Left = new IdToken($"Item{i + 1}");
```

Comment:
Strictly speaking `ItemN` name gets generated always. Won't it cause problems with cases like
```csharp
var data = (42, Payload: source());
pvf(data.Item2); // the same as pvf(data.Payload)
```
will it?